### PR TITLE
Added extraction of possible shortname from the dataset path, as well…

### PR DESCRIPTION
**Pull request by Jorgen-5**
… as modified dataset_state slightly

### CODE REVIEW

File: src/datadoc/backend/dapla_dataset_path_info.py [hunk 1]
```diff
@@ -406,3 +412,28 @@ def dataset_version(
             ):
                 return last_filename_element[1:]
         return None
+
+    @property
+    def dataset_shortname(
+        self,
+    ) -> str | None:
+        """Extract version information if exists in filename.
```
Comment (mmwinther) [inline src/datadoc/backend/dapla_dataset_path_info.py:420]: I think this is left over from a copy/paste?

Just update it to describe the information returned from this method. We could also mention that it is expected to be the part before the dataset_state based on Dapla's naming convention.

**Review by mmwinther — COMMENTED** [review]
The code looks good, just one little comment on the docstring

File: src/datadoc/backend/dapla_dataset_path_info.py (hunk 1 above)
Comment (Jorgen-5) [reply src/datadoc/backend/dapla_dataset_path_info.py]: Final commit adds the correct description 😊

**Review by mmwinther — APPROVED** [review]
Nice work!

**Pull request merged**